src/include/sof/audio/MaxxEffect/MaxxStream.h

```diff
@@ -0,0 +1,133 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This should be under `sof/src/include/sof/audio/codec_adapter/codec/MaxxEffect`.

Reply:
I wonder if maybe we should do a codec_adapter/codecs/<company> and in the kconfig just include a kconfig in the respective dir so all include info can be pushed down to individual kconfigs

Reply:
@cujomalainey 
I think this is a good idea but I prefer it to be handled in a separate PR.

Another thing I thought about is to upstream pass-through code for Waves codec.
It will come in handy to add SOF with Waves codec to CI checks.
Where is a place to discuss if this is needed?